### PR TITLE
feature/credits-page

### PR DESCRIPTION
**Pull request by ADI-projects30**
1. Create credits page with pictures, names and LinkedIn\Mail.
2. Create AUTHORS.md (I added some github names at the end of people who didn't give their names)

Names are ordered by alphabetic last name.

### CODE REVIEW

File: app/static/credits_style.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+body {
```
Comment (yammesicka) [inline app/static/credits_style.css:1]: Please reformat the CSS file using CSS Beautifier of some kind :)

File: app/templates/credits.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
+{% extends "base.html" %}
+
+{% block head %}
+{{ super() }}
+    <link href="{{ url_for('static', path='/credits_style.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+    <h1>Say hello to our developers:</h1>
+    <div class="gallery">
```
Comment (yammesicka) [inline app/templates/credits.html:10]: Prefer having this data in some kind of a JSON, and loading it using a for loop

File: AUTHORS.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+🐍 This document only reflects the team at the time of the original release in February 2021.
+
+# Our Authors
+
+Markup : * Yam Mesicka - Leader
```
Comment (yammesicka) [inline AUTHORS.md:5]: Remove the `Markup :`

**Review by yammesicka — COMMENTED** [review]
Great job! :) Let's make it perfect

File: app/static/credits_style.css (hunk 1 above)
Comment (ADI-projects30) [reply app/static/credits_style.css:1]: fixed :)

File: app/templates/credits.html (hunk 2 above)
Comment (ADI-projects30) [reply app/templates/credits.html]: fixed :)

File: AUTHORS.md (hunk 3 above)
Comment (ADI-projects30) [reply AUTHORS.md]: fixed :)

File: app/main.py [hunk 4]
```diff
@@ -42,7 +42,7 @@ def create_tables(engine, psql_environment):
 set_ui_language()
 
 from app.routers import (  # noqa: E402
-    agenda, calendar, categories, celebrity, currency, dayview,
+    agenda, calendar, categories, celebrity, currency, credits, dayview,
```
Comment (yammesicka) [inline app/main.py:45]: Reorder - credits should be before currency

File: app/static/credits_style.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+	text-align: center;
+}
+
+div.a.contact {
+	padding: 1em;
+	text-align: center;
+}
+
+p {
+	color: rgb(13, 110, 253)
```
Comment (yammesicka) [inline app/static/credits_style.css:34]: Don't forget the ;

File: app/routers/credits.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+from loguru import logger
+from starlette.templating import _TemplateResponse
+
+from app.dependencies import templates
+
+router = APIRouter()
+
+
+def data_from_json() -> List:
```
Comment (yammesicka) [inline app/routers/credits.py:14]: `credits_from_json` - Usually using the word `data` in variable is a red flag :)

File: app/routers/credits.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
+from fastapi import APIRouter, Request
+import json
+from typing import List
+
```
Comment (yammesicka) [inline app/routers/credits.py:4]: Remove one blank line

**Review by yammesicka — COMMENTED** [review]
Awesome! Few small additional fixes left

File: app/main.py (hunk 4 above)
Comment (ADI-projects30) [reply app/main.py]: fixed all the notes, ready for merge :)

File: app/static/credits_style.css (hunk 5 above)
Comment (ADI-projects30) [reply app/static/credits_style.css]: fixed :)

File: app/routers/credits.py (hunk 6 above)
Comment (ADI-projects30) [reply app/routers/credits.py]: fixed :)

File: app/routers/credits.py (hunk 7 above)
Comment (ADI-projects30) [reply app/routers/credits.py:4]: fixed :)

File: app/templates/credits.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+{% block head %}
+{{ super() }}
+    <link href="{{ url_for('static', path='/credits_style.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+    <h1>Say hello to our developers:</h1>
+    {% for credit in credit_list %}
+        <div class="gallery">
+            <img src="/static/credits_pictures/{{credit.picture}}" alt="{{credit.name}}">
```
Comment (yammesicka) [inline app/templates/credits.html:12]: Use `url_for`

File: app/templates/partials/index/navigation.html [hunk 9]
```diff
@@ -30,6 +30,9 @@
         <li class="nav-item">
           <a class="nav-link" href="/search">Search</a>
         </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/credits">Credits</a>
```
Comment (yammesicka) [inline app/templates/partials/index/navigation.html:34]: Prefer `url_for`

File: app/routers/credits.py [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+from loguru import logger
+from starlette.templating import _TemplateResponse
+
+from app.dependencies import templates
+
+router = APIRouter()
+
+
+def credits_from_json() -> List:
+    path = 'app/resources/credits.json'
```
Comment (yammesicka) [inline app/routers/credits.py:14]: Can you please define `RESOURCES` variable in the `config.py.example` and to use it instead?

I believe this line can do:

`RESOURCES_DIR = pathlib.Path(__file__).parent / 'resources'`

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/templates/credits.html (hunk 8 above)
Comment (ADI-projects30) [reply app/templates/credits.html:12]: Tried to use- url_for- before but it doesn't recognize the (credits.picture), I got an error that there is no picture key, and that is why I did it like that, is it ok ?

Comment (yammesicka) [reply app/templates/credits.html:12]: Search url_for in this project, there are solutions to it :)

File: app/routers/credits.py (hunk 10 above)
Comment (ADI-projects30) [reply app/routers/credits.py]: fixed :)

File: app/templates/partials/index/navigation.html (hunk 9 above)
Comment (ADI-projects30) [reply app/templates/partials/index/navigation.html]: fixed :)

File: app/templates/credits.html (hunk 8 above)
Comment (ADI-projects30) [reply app/templates/credits.html]: fixed all the notes, ready for merge :)

File: requirements.txt [hunk 11]
```diff
@@ -83,6 +83,7 @@ nodeenv==1.5.0
 outcome==1.1.0
 packaging==20.8
 passlib==1.7.4
+pathlib==1.0.1
```
Comment (yammesicka) [inline requirements.txt:86]: Remove

File: app/main.py [hunk 12]
```diff
@@ -73,6 +73,7 @@ async def swagger_ui_redirect():
     categories.router,
     celebrity.router,
     currency.router,
+    credits.router,
```
Comment (yammesicka) [inline app/main.py:76]: ABC

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/templates/base.html [hunk 13]
```diff
@@ -54,6 +54,9 @@
                         <li class="nav-item">
                             <a class="nav-link" href="{{ url_for('search') }}">Search</a>
                         </li>
+                        <li class="nav-item">
+                          <a class="nav-link" href="/credits">Credits</a>
```
Comment (yammesicka) [inline app/templates/base.html:58]: `url_for`

File: app/templates/credits.html [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+    <link href="{{ url_for('static', path='/credits_style.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+    <h1>Say hello to our developers:</h1>
+    {% for credit in credit_list %}
+        <div class="gallery">
+            <img src="{{ url_for('static', path='/credits_pictures/' + credit.picture + '.PNG') }}" alt="{{credit.name}}">
+            <div class="credit-name">{{credit.name}}</div>
+            <div class="credit-name">
```
Comment (yammesicka) [inline app/templates/credits.html:14]: Maybe `credit-details`?

File: app/main.py (hunk 12 above)
Comment (ADI-projects30) [reply app/main.py]: fixed all the notes, ready for merge :)

File: app/templates/base.html (hunk 13 above)
Comment (ADI-projects30) [reply app/templates/base.html]: fix :)

File: app/templates/credits.html (hunk 14 above)
Comment (ADI-projects30) [reply app/templates/credits.html]: fix :)

File: requirements.txt (hunk 11 above)
Comment (ADI-projects30) [reply requirements.txt]: fix :)

**Review by yammesicka — APPROVED** [review]

**Pull request merged**